x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/findings_by_resource_container.tsx

```diff
@@ -97,10 +101,28 @@ const LatestFindingsByResource = ({ dataView }: FindingsBaseProps) => {
           />
         </PageTitle>
         {error && <ErrorCallout error={error} />}
-
         {!error && (
           <>
             <FindingsGroupBySelector type="resource" />
+            {findingsGroupByResource.isSuccess && !!findingsGroupByResource.data.page.length && (
+              <FindingsDistributionBar
+                {...{
+                  type: i18n.translate(
```

Comment:
This ID feels wrong (missing a `.findings` section for instance) please make sure it is algined with @JordanSh work regarding i18n IDs in this page